docs/cache.md

```diff
@@ -161,7 +161,7 @@ import { SlidingCache } from "bee-agent-framework/cache/slidingCache";
 
 const cache = new SlidingCache<number>({
   size: 3, // (required) number of items that can be live in the cache at a single moment
-  ttl: 1000, // (optional, default is Infinity) Time in miliseconds after the element is removed from a cache
+  ttl: 1000, // (optional, default is Infinity) Time in milliseconds after the element is removed from a cache
```

Comment:
This file is automatically generated from `bee-agent-framework/cache/slidingCache` so you have to update that file first.

Reply:
Correct me if I'm wrong this is a *spelling mistake in a comment* so I don't think the original file that has the class definition (whose constructor is called here) is relevant here

Reply:
The content of the original source code file is copied into this markdown file. Hence, you should modify the original source code file; otherwise, it will be overridden.

-> update comment in `bee-agent-framework/cache/slidingCache.ts` and run `yarn docs:build`